app/views/layouts/_header.html.erb

```diff
@@ -27,12 +22,33 @@
                     <li><a href="#">Bookings Received</a></li>
                   </ul>
                 </li>
+            </ul>
+            <ul class="nav navbar-nav navbar-right">
+                <!-- <li><%= link_to "About Us", static_pages_help_path %></li> -->
+                <li><%= link_to "FAQ",  static_pages_faq_path %></li>
+                <li><%= link_to "How It Works?", root_path(:anchor => "how_it_works") %></li>
                 <% if current_user.present? %>
-                <li><%= link_to current_user.username, static_pages_dashboard_path %></li>
-                <li><%= link_to('Logout', destroy_user_session_path, :method => :delete) %></li>
+                <li>
+                <%= link_to static_pages_dashboard_path do %>
+                    <span class="glyphicon glyphicon-user" aria-hidden="true"></span> <%= current_user.username %>
+                <% end %>
+                </li>
+                <li>
+                <%= link_to destroy_user_session_path, :method => :delete do %>
+                    <span class="glyphicon glyphicon-log-out" aria-hidden="true"></span> Logout
+                <% end %>
+                </li>
                 <% else %>
-                <li><%= link_to "Sign Up", new_user_registration_path %></li>
-                <li><%= button_to "Sign In", new_user_session_path, :class => "btn btn-default navbar-btn", :method => :get %></li>
+                <li>
+                <%= link_to new_user_registration_path do %>
+                    <span class="glyphicon glyphicon-new-window" aria-hidden="true"></span> Sign up
+                <% end %>
+                </li>
+                <li>
+                <%= link_to new_user_session_path, :class => "btn btn-default" do %>
```

Comment:
This change made the Sign In button a little bit higher, increasing main navbar's height. It feels weird, because when you sign in and that button is no longer there, navbar's height decreases again.
There's also a problem with the button color, which disappears when you hover over it.

I tried to solve this by adding the class `navbar-btn` but that wasn't very helpful. It was only after reverting `link_to` to `button_to` that I was able to correct this. Is there any reason to switch this from a button to a link?